Source/Scene/FrameState.js

```diff
@@ -156,6 +156,14 @@ define([
          */
         this.maximumScreenSpaceError = undefined;
 
+        /** The factor at which all Screen Space Error is scaled. This factor accounts for a difference in screen
+         * density with resolution scaling to decouple SSE from effective resolution.
```

Comment:
Fix the formatting of the comment:

```js
        /**
         * The factor at which all Screen Space Error is scaled. This factor accounts for a difference in screen
         * density with resolution scaling to decouple SSE from effective resolution.
```

Reply:
This description is slightly different than the one in `Scene`. For consistency make them both the same.